index.html

```diff
@@ -18,8 +18,20 @@
     <a class="item active" href="index.html">
         Home 
     </a>
-    <a class="item" href="./pages/about.html">
-        About
+    <a class="item"></a>
+        <div class="ui simple dropdown item">
+            About
+            <i class="dropdown icon"></i>
+            <div class="menu">
+                <div class="item" href="./pages/about/principal.html"> Principal's Welcome</div>
```

Comment:
If you change these `div`s to `a`s you should be able to click through to the pages :+1:

Reply:
ty

Reply:
In terms of the visual bug, try remove the closing `</a>` on line 20: https://github.com/dominikkawka/SchoolWebsite/blob/69fcdf363784a1a90c04c87170884f1892df1112/index.html#L20
since you already close it on line 34: https://github.com/dominikkawka/SchoolWebsite/blob/69fcdf363784a1a90c04c87170884f1892df1112/index.html#L34
that might fix it

Reply:
Actually, I don't think you need that extra ` <a class="item">` at all, since the dropdown div already has the `.item` css class.

Reply:
I did that, it didn't fix the visual bug and it caused the navbar to go fat dosa
![woo](https://user-images.githubusercontent.com/31955648/38929475-54be2e60-4304-11e8-8b8c-e195227ff51a.png)

Reply:
![bean](https://user-images.githubusercontent.com/31955648/38929494-648c296e-4304-11e8-93ae-7afc6d9eb35d.png)

Reply:
I got rid of the <a class item>, it fixed the weird gap between home and about

Reply:
yeah it's fixed now

Reply:
![uwu](https://user-images.githubusercontent.com/31955648/38929683-0d48212a-4305-11e8-96c4-69923d37047b.png)
![owo](https://user-images.githubusercontent.com/31955648/38929685-1294da10-4305-11e8-881a-bffd15c637f2.png)